Makefile

```diff
@@ -49,7 +32,9 @@ install: all
 dist: clean
 	rm -rf $(PACKAGE)
```

Comment:
Maybe use `git archive --format=tar --prefix=$(PACKAGE)/ | bzip2 >$(PACKAGE).tar.bz2` for this?